src/frontend/data/sideEffects/endLive/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { API_ENDPOINT } from '../../../settings';
+import { Video } from '../../../types/tracks';
+import { appData } from '../../appData';
+
+export const endLive = async (video: Video) => {
+  const response = await fetch(`${API_ENDPOINT}/videos/${video.id}/end-live/`, {
```

Comment:
Shouldn't we migrate from sideEffects conventions to those using react-query ?

Reply:
I don't see the interest in that case. I don't want to do cache not poll this endpoint. It's just a simple fetch I don't think I need react-query here.